core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -161,15 +217,38 @@ private CodeSystem getCodeSystemById(String shortNameOrOid) {
 				.getSync(1, TimeUnit.MINUTES);
 	}
 	
-	private String getCodeSystemVersionInformation(CodeSystemVersion codeSystemVersion) {
+	//I added the CodeSystem argument so i don't have to search for it in a request. It saves a lot of time
+	private String getCodeSystemVersionInformation(CodeSystemVersion codeSystemVersion, CodeSystem codeSystem) {
 		return new StringBuilder()
-			.append("\tVersion id: ").append(codeSystemVersion.getVersion()).append("\n")
-			.append("\tDescription: ").append(codeSystemVersion.getDescription()).append("\n")
-			.append("\tEffective date: ").append(EffectiveTimes.format(codeSystemVersion.getEffectiveTime(), DateFormats.DEFAULT)).append("\n")
-			.append("\tCreation date: ").append(Dates.formatByHostTimeZone(codeSystemVersion.getImportDate(), DateFormats.DEFAULT)).append("\n")
-			.append("\tLast update: ").append(codeSystemVersion.getLastModificationDate() != null ? StdDateFormat.getDateInstance().format(codeSystemVersion.getLastModificationDate()) : "-").append("\n")
-			.append("\tVersion branch path: ").append(codeSystemVersion.getPath())
+			.append("\t\tVersion id: ").append(codeSystemVersion.getVersion()).append("\n")
+			.append("\t\tDescription: ").append(codeSystemVersion.getDescription()).append("\n")
+			.append("\t\tEffective date: ").append(EffectiveTimes.format(codeSystemVersion.getEffectiveTime(), DateFormats.DEFAULT)).append("\n")
+			.append("\t\tCreation date: ").append(Dates.formatByHostTimeZone(codeSystemVersion.getImportDate(), DateFormats.DEFAULT)).append("\n")
+			.append("\t\tLast update: ").append(codeSystemVersion.getLastModificationDate() != null ? StdDateFormat.getDateInstance().format(codeSystemVersion.getLastModificationDate()) : "-").append("\n")
+			.append("\t\tVersion branch path: ").append(codeSystemVersion.getPath()).append("\n")
+			.append("\t\tExtensions: ").append(getExtensionsForGivenVersionOfCodeSystem(codeSystemVersion, codeSystem))
+			.append("\n")
 			.toString();
 	}
 	
+	private String getExtensionsForGivenVersionOfCodeSystem(CodeSystemVersion version, CodeSystem extendedCodeSystem) {
+		final List<String> extensionsOfThisVersion =
+		CodeSystemRequests
+		.getAllCodeSystems(getContext())
```

Comment:
A filter would be great here to reduce the number of Code Systems resources returned.

Reply:
it does not have a filterBy() method 😥

Reply:
@cmark : Ignore